io/io/src/TFile.cxx

```diff
@@ -383,6 +383,14 @@ TFile::TFile(const char *fname1, Option_t *option, const char *ftitle, Int_t com
    if (fOption == "NEW")
       fOption = "CREATE";
 
+   if (fOption == "AVOIDGLOBALREGISTRATION") {
```

Comment:
```suggestion
   if (fOption == "SKIPGLOBALREGISTRATION") {
```

Reply:
This name was suggested by @Axel-Naumann earlier in the discussion.  I have no strong preference.

Reply:
Ok, changed names and swapped logic as suggested.